.github/workflows/deploy.yml

```diff
@@ -27,6 +27,7 @@ jobs:
         run: |
           if [[ ${{ github.ref }} == refs/heads/main ]] || [[ "${{github.event.head_commit.message}}" == *"[testnetdeploy]"* ]]; then
             echo "::set-output name=deployment::testnet" 
+            echo "::set-output name=vm_host::${{ secrets.TESTNET_VM_HOST }}"
```

Comment:
The new secret was created in the repo; so on `main` merge or `[testnetdeploy]` that VM will be targeted in the deployment action.